README.md

```diff
@@ -0,0 +1,68 @@
+Configuration files for Intel AudioVoiceSpeech (AVS) driver
+========================
+
+ * [Overview](#overview)
+   * [DMIC](#dmic)
+   * [HDAudio](#hdaudio)
+   * [HDMI](#hdmi)
+   * [I2S](#i2s)
+
+# Overview
+
+Each configuration file represents an audio stream topology i.e.: connections of AudioDSP pipelines
+and processing modules that provide rich user experience with the Intel AVS stack. The avs-driver is
+part of the Linux kernel sound subsystem,
+[sound/soc/intel/avs](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/sound/soc/intel/avs).
+
+Mimics [alsa-topology-conf](https://github.com/alsa-project/alsa-topology-conf) yet instead
+of UseCaseManagement (UCM) files, topologies found here are XML document files.
+
+As the audio stack supports audio transfer over multiple interfaces, repository consists of
+directories representing one interface each.
+
+Kernel sound drivers are not capable of consuming topologies in their XML format. In order to obtain
+kernel-friendly format, topology needs to be converted to UCM format first using
+[avstplg](https://github.com/thesofproject/avsdk/tree/main/avstplg) tool and then finaly to binary
```

Comment:
Eh... found one more typo: finaly -> finally

Reply:
No problem fixing that too.